pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "hatchling.build"
 [project]
 name = "omg_dosimetry"
 
-version = "1.5.5"
+version = "1.6.0"
```

Comment:
So, from what I understand: is this the version line that has to be updated each pull request?

Reply:
Correct. Pull requests to Master should have the version number incremented. We're using the recommended "semantic versioning" (https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#semantic-versioning-preferred).

We implemented automatic actions that are triggered with every pull request to master. Documentation on ReadTheDocs is updated automatically, and the new version is built and published on pypi.